Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBSDKKeychain.m

```diff
@@ -67,8 +67,7 @@ + (NSString *)retrieveTokenWithKey:(NSString *)key {
 }
 
 + (NSArray<NSString *> *)retrieveAllTokenIds {
-  NSMutableDictionary<id, id> *query = [DBSDKKeychain
-      queryWithDict:@{(id)kSecReturnAttributes : (id)kCFBooleanTrue, (id)kSecMatchLimit : (id)kSecMatchLimitAll}];
+  NSMutableDictionary<id, id> *query = [DBSDKKeychain queryWithDict:@{(id)kSecReturnAttributes : (id)kCFBooleanTrue, (id)kSecMatchLimit : (id)kSecMatchLimitAll}];
```

Comment:
Ah ha, I see. No problem there. I tend to let Xcode wrap my source for me instead of doing it manually.